content/en/continuous_integration/tests/dotnet.md

```diff
@@ -148,6 +152,26 @@ if (scope != null) {
 
 To create filters or `group by` fields for these tags, you must first create facets. For more information about adding tags, see the [Adding Tags][7] section of the .NET custom instrumentation documentation.
 
+### Adding custom metrics to tests
+
+<div class="alert alert-warning">
+  <strong>Note:</strong> To add custom metrics to tests, you must configure <a href="#custom-instrumentation">Custom Instrumentation</a> first.
+</div>
+
+Just like tags, you can add custom metrics to your tests by using the current active span:
+
+```csharp
+// inside your test
+var scope = Tracer.Instance.ActiveScope; // from Datadog.Trace;
+if (scope != null) {
+    scope.Span.SetTag("memory_allocations", 16);
+}
+// test continues normally
+// ...
+```
+
+To create filters or `group by` fields for these tags, you must first create facets. For more information about adding tags, see the [Adding Tags][7] section of the .NET custom instrumentation documentation.
```

Comment:
```suggestion
To create filters or visualizations for these tags, you must first create facets. For more information about adding tags, see the [Adding Tags][7] section of the .NET custom instrumentation documentation.
```
I don't think you can group by a measure 

Also, here we should link to the guide here https://github.com/DataDog/documentation/pull/18831

Reply:
https://github.com/DataDog/documentation/pull/18831 is merged, so you can link to the guide